src/ruuvitag_ble/df5_decoder.py

```diff
@@ -75,7 +94,3 @@ def movement_counter(self) -> int:
     @property
     def measurement_sequence_number(self) -> int:
         return self.data[9]
-
-    @property
-    def mac(self) -> str:
-        return ":".join(f"{x:02X}" for x in self.data[10:])
```

Comment:
Why does this get removed?

Reply:
I didn't see much point in extracting this from the advertisement when it's already available in the service data.  I don't think it's possible for them to be different.  This goes with the removal of its use in the parser.